### PR TITLE
chore: Displays the dataset description in a tooltip in the datasets list

### LINKED ISSUE #16309: [listview]add dataset description to dataset listview on hovering
**Is your feature request related to a problem? Please describe.**
The dataset name is too unfriendly for non-English users!!!

**Describe the solution you'd like**
The dataset name is too unfriendly for non-English users, I hope to add a "dataset display name " field, allowing users to modify according to their own language

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by michael-s-molina**
### SUMMARY
Closes https://github.com/apache/superset/issues/16309

@junlincc @jinghua-qa @rusackas @zhxjdwh @mistercrunch 

### AFTER SCREENSHOTS OR ANIMATED GIF
https://user-images.githubusercontent.com/70410625/130331481-ccc65d2f-9680-48d6-a3c7-2cc7878c0baa.mov

### TESTING INSTRUCTIONS
Check the video for instructions.

### ADDITIONAL INFORMATION
- [x] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/src/components/InfoTooltip/index.tsx [hunk 1]
```diff
@@ -52,6 +52,14 @@ const StyledTooltip = styled(Tooltip)`
   }
 `;
 
+const StyledTooltipTitle = styled.span`
+  display: -webkit-box;
```
Comment (kgabryje) [inline superset-frontend/src/components/InfoTooltip/index.tsx:56]: Why do we use webkit box instead of flex?

Comment (kgabryje) [reply superset-frontend/src/components/InfoTooltip/index.tsx:56]: Nevermind, I've just read https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-line-clamp 👍

**Review by kgabryje — APPROVED** [review]
Lgtm!

**Review by villebro — APPROVED** [review]
Very nice, LGTM!

**Comment by zhxjdwh** [comment]
i don't think this is good solution!!!!!!

**Comment by rusackas** [comment]
> i don't think this is good solution!!!!!!

Could you elaborate as to why, or what alternate approach you would suggest?

**Comment by zhxjdwh** [comment]
# I think we should add a field "display name" for dataset/table  , and add a column for "display name" field in datasets list page!

**Comment by zhxjdwh** [comment]
@rusackas 
I think we should add a field "display name" for dataset/table , and add a column for "display name" field in datasets list page!

**Comment by michael-s-molina** [comment]
> i don't think this is good solution!!!!!!
> I think we should add a field "display name" for dataset/table , and add a column for "display name" field in datasets list page!

@zhxjdwh @rusackas @junlincc I think we need to separate the features a little bit:
- One feature, which is the objective of this PR, is to present the description of the datasets in the list. Similar to what we have when creating a chart.
- Another feature is to present an alternative name or display name for a dataset when the name is unfriendly for non-English users. Here I'm inclined to agree with @zhxjdwh that the use of the description is not the ideal solution. The user would be forced to hover on each dataset until he finds what he was looking for. We also need to consider that the solution is not only adding a column to the list with the alternative name. We need to change all the places where the dataset is displayed to show the display name instead of the original name. This includes Explore, Add Chart, searching, etc.

**Pull request merged**